BUILDING.md

```diff
@@ -4,12 +4,23 @@ This guide is useful if you intend to contribute on containerd. Thanks for your
 effort. Every contribution is very appreciated.
 
 This doc includes:
+* [Getting started with GitHub Codespaces](#getting-started-with-gitHub-codespaces)
 * [Build requirements](#build-requirements)
 * [Build the development environment](#build-the-development-environment)
 * [Build containerd](#build-containerd)
 * [Via docker container](#via-docker-container)
 * [Testing](#testing-containerd)
 
+## Getting started with GitHub Codespaces
+
+To get started, create a codespace for this repository by clicking this 👇 
```

Comment:
This line has a trailing white space. That's why the CI is complaining.

Reply:
Ohh! It seems it checks each individual commit for white space :smile:. You may need to squash.

Reply:
Also, all commits need a valid DCO. This one does not have one: https://github.com/containerd/containerd/pull/8295/commits/0a28da599c71b8f221bb0d9cf9508c919f50bf55

Reply:
I see, thanks! Updated,  looks like the CI test passed this time